ui-src/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
what is this file for?
looks like an IDE config file. let's add this to .gitignore if so and not check it in.

Reply:
The file is for spacing and linting and should be checked in order to ensure consistency.